data/test-segmentio.sh

```diff
@@ -2,21 +2,68 @@
 accessKey=$1
 
 # normal case
-curl -i -X POST http://localhost:7070/webhooks/segmentio.json?accessKey=$accessKey \
--H "Content-Type: application/json" \
--d '{
-  "version"   : 1,
-  "type"      : "identify",
-  "userId"    : "019mr8mf4r",
-  "traits"    : {
-      "email"            : "achilles@segment.com",
-      "name"             : "Achilles",
-      "subscriptionPlan" : "Premium",
-      "friendCount"      : 29
-  },
-  "timestamp" : "2012-12-02T00:30:08.276Z"
-}' \
--w %{time_total}
+curl -H "Accept: application/json; version=2.0" \
+     http://spec.segment.com/generate/identify | \
+curl -X POST \
+    -H "Content-Type: application/json" \
+    -d @- \
+    http://localhost:7070/webhooks/segmentio.json?accessKey=$accessKey
+
```

Comment:
i tried running this, but seems that http://spec.segment.com/ will return different data each time and sometimes i got "{"message":"Supported segment.io API versions: [2]"}" is it expected?
Btw, would be great if you could print the received version in the error message too.